### PR TITLE
Review html page names in Core docs - closes #657

### LINKED ISSUE #657: Review HTML page names in Core docs
Review, if the file names of the docs pages can be improved.
example: API specification has api.html, configuration has config.html, Create custom transaction types has customize.html, reference to references.

 Review file names in latest Core docs

This issue is linked to the epic parent issue #532

### PR DESCRIPTION
**Pull request by chris529**
Closes issue #657

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 * xref:setup/index.adoc[Setup]
-** xref:setup/Application.adoc[Application]
-** xref:setup/commander.adoc[Commander application]
-** xref:setup/docker.adoc[Docker image]
-** xref:setup/source.adoc[Source code]
+** xref:setup/application.adoc[Application]
+** xref:setup/commander-application.adoc[Commander application]
+** xref:setup/docker-image.adoc[Docker image]
+** xref:setup/source-code.adoc[Source code]
 * xref:management/index.adoc[Management]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc:8]: I think we discussed the html names of the different distributions when we were renaming them, and agreed to leave them with the short name.

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -1,15 +1,15 @@
 
 * xref:interact-with-network.adoc[Interact with the network]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: This name could be improved to `interact-with-the-api.adoc`.

If you do, please think of updating the title of the page as well.

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+** xref:setup/source-code.adoc[Source code]
 * xref:management/index.adoc[Management]
-** xref:management/application.adoc[Application commands]
-** xref:management/commander.adoc[Commander application commands]
-** xref:management/docker.adoc[Docker image commands]
-** xref:management/source.adoc[Source code commands]
+** xref:management/application-commands.adoc[Application commands]
+** xref:management/commander-commands.adoc[Commander application commands]
+** xref:management/docker-commands.adoc[Docker image commands]
+** xref:management/source-commands.adoc[Source code commands]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: I would not add `-commands` here. 

To make it consistent, I suggest to change the titles in `management` from e.g. `Docker commands` to `Docker image`

```suggestion
** xref:management/application.adoc[Application]
** xref:management/commander.adoc[Commander application]
** xref:management/docker.adoc[Docker image]
** xref:management/source.adoc[Source code]
```

File: modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 * xref:interact-with-network.adoc[Interact with the network]
 * xref:setup/index.adoc[Setup]
-** xref:setup/Application.adoc[Application]
-** xref:setup/commander.adoc[Commander application]
-** xref:setup/docker.adoc[Docker image]
-** xref:setup/source.adoc[Source code]
+** xref:setup/application.adoc[Application]
+** xref:setup/commander-application.adoc[Commander application]
+** xref:setup/docker-image.adoc[Docker image]
+** xref:setup/source-code.adoc[Source code]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc:7]: I think we discussed the html names of the different distributions when we were renaming them, and agreed to leave them with the short name.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please see my comments!

File: modules/ROOT/nav.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: Agreed...now done 👍

File: modules/ROOT/nav.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/nav.adoc:7]: OK, I think I should have caught all the associated links related to the page name changes etc. :-)

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**